Palo Alto Networks/paloalto/_meta/fields.yml

```diff
@@ -0,0 +1,24 @@
+action.target:
+  description: 'Target action'
+  name: action.target
+  type: keyword
+geo.country_iso_code:
+  description: "Country ISO code"
+  name: geo.country_iso_code
+  type: keyword
+http.response.content_type:
+  description: "Http reponse content type"
+  name: http.response.content_type
+  type: keyword
+action.properties.subject:
+  description: "Subject"
+  name:  action.properties.subject
+  type: keyword
+email.from:
```

Comment:
`email.from.address` exists now in ECS, but I don't know since which version

Reply:
I modified it